augmentedreality/Common/include/LocationDataSource.h

```diff
@@ -34,7 +34,7 @@ class LocationDataSource : public QObject
   Q_PROPERTY(QGeoPositionInfoSource* geoPositionSource READ geoPositionSource
              WRITE setGeoPositionSource NOTIFY geoPositionSourceChanged)
   Q_PROPERTY(QCompass* compass READ compass WRITE setCompass NOTIFY compassChanged)
-  Q_PROPERTY(bool started READ isStarted NOTIFY isStartedChanged)
+  Q_PROPERTY(bool isStarted READ isStarted NOTIFY isStartedChanged)
```

Comment:
had to change the property because qdocs was expecting this name convention from the `isStarted()` function. I don't think there should be any problems, since the property `isStarted` is accessed via the getter. It doesn't matter the name of it

Reply:
❌ This is a breaking change which changes the API interface of the QML API. It goes against the QML naming conventions.

I'm not sure what the problem is that we are trying to solve here, but since none of the properties are documented in QML, it might instead be resolved by documenting these propertites?

Reply:
⁉"_It goes against the QML naming conventions._" The above properties have the same name as their getter.

💭I think qdoc is internally calling a linking (\l) based on the getter name. With the property started, there is an error about missing started property, which is not using isStarted (I guess qdoc is automatically document the property).

I will try to manually document the property.

Reply:
💭  The second parameter of `Q_PROPERTY` is the name of the property in the QML API, all other arguments are to do with the behaviour of the property and are transparent to the QML interface (there is no way to call `isStarted()` from QML unless it is marked with `Q_INVOKABLE`).